apps/web/pages/settings/my-account/out-of-office/index.tsx

```diff
@@ -39,18 +40,58 @@ export type BookingRedirectForm = {
   toTeamUserId: number | null;
   reasonId: number;
   notes?: string;
+  uuid?: string | null;
+};
+
+export type outOfOfficeEntryData = {
+  uuid: string;
+  selectedReason: number | null;
+  profileRedirect: boolean;
+  selectedMember: number | null;
+  notes: string | null;
+  start: Date;
+  end: Date;
 };
 
 const CreateOutOfOfficeEntryModal = ({
```

Comment:
Should be `CreateOrUpdateOutOfOfficeEntryModal`. In fact, everything needs to be changed from `create` to `createOrUpdate`. Please go through the code thoroughly and make the necessary changes

Reply:
Got it.. will do it..